### PR TITLE
Neural networks

### PR DESCRIPTION
**Pull request by pasliwa**

### CODE REVIEW

**Comment by magickris93** [comment]
@pasliwa 

- [ ] It would be a good idea to automate selection of column names. It may be problematic if input data set changes.
- [ ] Make sure your code is properly indented. For example :

```python
for el in possible_layer_combinations:
    clf = MLPClassifier(solver='lbfgs', alpha=1e-5,
                    hidden_layer_sizes=el, random_state=1)
```

- [ ] You should get rid of empty cells.

**Comment by rlatawiec** [comment]
- [ ] Make your code PEP8 compliant (mostly lines that are too long)
- [ ] Add more validation metrics, accuracy alone doesn't paint the whole picture
- [ ] I'd use a premade method for parameter tuning like `RandomizedSearchCV` from `sklearn.model_selection`
